pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>org.primeframework</groupId>
   <artifactId>prime-mvc</artifactId>
-  <version>4.22.4</version>
+  <version>4.22.7</version>
```

Comment:
You should also update `build.savant`

Reply:
Yes, was waiting to confirm this was legit first